src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -469,9 +469,46 @@ public virtual RedirectToActionResult RedirectToAction(string actionName, string
             string controllerName,
             object routeValues)
         {
-            return new RedirectToActionResult(actionName, controllerName, routeValues)
+            return RedirectToAction(actionName, controllerName, routeValues, fragment: null);
+        }
+
+        /// <summary>
+        /// Redirects to the specified action using the specified <paramref name="actionName"/>,
+        /// <paramref name="controllerName"/>, and <paramref name="fragment"/>.
+        /// </summary>
+        /// <param name="actionName">The name of the action.</param>
+        /// <param name="controllerName">The name of the controller.</param>
+        /// <param name="fragment">The fragment to add to the URL</param>
```

Comment:
Please add a period at the end of this comment. Same for the `fragment` doc comment in the overload just below.